modules/apps/frontend-taglib/frontend-taglib-sample-web/src/main/resources/META-INF/resources/js/ClaySampleMultiselectPropsTransformer.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayDropDown from '@clayui/drop-down';
+import ClayIcon from '@clayui/icon';
+import ClaySticker from '@clayui/sticker';
+import {getSpritemap} from '@liferay/frontend-icons-web';
+import React from 'react';
+
+const spritemap = getSpritemap();
+
+const MenuCustom = ({
```

Comment:
What's the purpose of all the code in `/js` dir? Isn't just `<liferay-ui:search-paginator>` in JSP enough for this task?

Reply:
js files can be removed, because they are not needed. but should we keep the js project structure? I mean `tsconfig, package.json`, etc. files for future js additions?

Reply:
I've removed the whole js structure, as is not required for this PR